files/expected-outputs/hello-world-ascii.json

```diff
@@ -1 +0,0 @@
-../inputs/hello-world-ascii.json
```

Comment:
For some reason CMake + MSVS didn't like the symbolic links. Problems were happening when copying during configure into the test directories. I just removed the symbolic link and duplicated the file. Within git this does not take up any extra space since git is content addressable, however, you must take care that these files always remain the same.